src/backend/package.json

```diff
@@ -42,10 +45,14 @@
     "@types/supertest": "2.0.11",
     "@typescript-eslint/eslint-plugin": "5.0.0",
     "@typescript-eslint/parser": "5.0.0",
-    "eslint": "8.0.1",
+    "eslint": "8.24.0",
```

Comment:
I firmly believe linting and other development tools must be installed globally in your system and not in an API folder.  A suggestion is to set up a bash script and place it at the root and create a make command that runs it
localtools.sh
 npm install -g eslint
 npm install -g eslint-config-prettier etc
etc

Let us be cautious that these dependencies can cause build fails during version updates. If this can be somehow automated at deployment time or setup a script that installs the necessary tools for local development that would be better.

Reply:
we can discuss this issue next Monday

Reply:
This is discussed separately with you @antsand and as team of team discussion. For now, we will go ahead with the husky implementation to set the foundation early on in this project and we will revisit this later in retros or if it create issues. Marking it resolved.